build-tools/packages/build-cli/src/test/commands/generate/assertTags.test.ts

```diff
@@ -0,0 +1,145 @@
+			const configData = result.config as {
+				assertionFunctions: Record<string, number>;
+			};
+			assert.strictEqual(configData.assertionFunctions.assert, 1);
+			assert.strictEqual(configData.assertionFunctions.fail, 0);
+		});
+
+		it("loads .cjs config files", async () => {
+			const testDir = await createTestFixture("assertTagging.config.cjs");
+
+			const config = lilconfig(configName, {
+				searchPlaces: [`${configName}.config.cjs`],
+			});
+
+			const result = await config.search(testDir);
+
+			assert(result !== null, "Config should be found");
+			assert("assertionFunctions" in result.config, "Config should have expected structure");
+			const configData = result.config as {
+				assertionFunctions: Record<string, number>;
+			};
+			assert.strictEqual(configData.assertionFunctions.assert, 1);
+			assert.strictEqual(configData.assertionFunctions.fail, 0);
+		});
+
+		it("loads .mjs config with empty assertionFunctions", async () => {
+			const testDir = await createTestFixture("assertTagging-empty.config.mjs");
+
+			const config = lilconfig(configName, {
+				searchPlaces: [`${configName}-empty.config.mjs`],
+			});
+
+			const result = await config.search(testDir);
+
+			assert(result !== null);
+			const configContentParsed = result.config as {
+				assertionFunctions: Record<string, number>;
+			};
+			assert(
+				typeof configContentParsed.assertionFunctions === "object",
+				"assertionFunctions should be an object",
+			);
+			assert(
+				Object.keys(configContentParsed.assertionFunctions).length === 0,
+				"assertionFunctions should be empty (disables tagging)",
+			);
+		});
+
+		it("returns null when no config file exists", async () => {
+			const testDir = await mkdtemp(path.join(tmpdir(), "assertTagging-no-config-"));
+			testDirs.push(testDir);
+
+			const config = lilconfig(configName, {
+				searchPlaces: [`${configName}.config.mjs`],
+			});
+
+			const result = await config.search(testDir);
+
+			// Should return null when no config is found
+			assert(
+				result === null || result === undefined,
+				"Should return null when no config exists",
+			);
+		});
+
+		it("prefers .cjs over .mjs when both exist", async () => {
+			const testDir = await mkdtemp(path.join(tmpdir(), "assertTagging-test-"));
+			testDirs.push(testDir);
+
+			// Copy both config files
+			await copyFile(
+				path.join(fixturesDir, "assertTagging.config.cjs"),
+				path.join(testDir, "assertTagging.config.cjs"),
+			);
+			await copyFile(
+				path.join(fixturesDir, "assertTagging.config.mjs"),
+				path.join(testDir, "assertTagging.config.mjs"),
+			);
+
+			const config = lilconfig(configName, {
+				searchPlaces: [`${configName}.config.cjs`, `${configName}.config.mjs`],
+			});
+
+			const result = await config.search(testDir);
+
+			assert(result !== null, "Config should be found");
+			assert(result.filepath.endsWith(".cjs"), "Should load .cjs file when listed first");
```

Comment:
Is it because it was listed first in `searchPlaces` or because both exist, like the test title says?